docs/development/coding-style.md

```diff
@@ -39,9 +39,16 @@ etc.
   * [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
     instead of string concatenation using `+`
 
-## API Names
+## Naming Things
 
-When creating a new API, we should prefer getters and setters instead of
+Electron APIs uses the same capitalization scheme as Node.js:
+
+- When the module itself is a class like `BrowserWindow`, use CamelCase.
+- When the module is a set of APIs, like `clipboard`, use mixedCase.
```

Comment:
Maybe `globalShortcut` is a better example here than `clipboard` since it is `mixedCase` itself.